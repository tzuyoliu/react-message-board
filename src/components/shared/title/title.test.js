import React from 'react'
import { create } from 'react-test-renderer'
import Title from '~/components/shared/title'

describe('title.test.js', () => {
    test('<Title /> should render', () => {
        const title = create(<Title />)
        expect(title).toBeTruthy()
    })

    test('<Title /> should match the snapshot', () => {
        const title = create(<Title>Snapshot Test</Title>)
        expect(title.toJSON()).toMatchSnapshot()
    })
})
