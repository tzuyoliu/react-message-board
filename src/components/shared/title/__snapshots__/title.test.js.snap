// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title.test.js <Title /> should match the snapshot 1`] = `
<h1
  className="sc-bdVaJa cSUaSj"
>
  Snapshot Test
</h1>
`;
